pkg/controllers/tunnel/tunnel.go

```diff
@@ -28,8 +28,8 @@ type TunnelController struct {
 	endpointWorkqueue workqueue.RateLimitingInterface
 }
 
-func NewTunnelController (objectNamespace string, ce cableengine.CableEngine, kubeClientSet kubernetes.Interface, submarinerClientSet submarinerClientset.Interface, endpointInformer submarinerInformers.EndpointInformer) *TunnelController {
-	tunnelController := &TunnelController{
+func NewTunnelController(objectNamespace string, ce cableengine.CableEngine, kubeClientSet kubernetes.Interface, submarinerClientSet submarinerClientset.Interface, endpointInformer submarinerInformers.EndpointInformer) *Controller {
```

Comment:
Similar to removing "Tunnel" from the struct name, we can avoid stutter here as well by renaming to NewController or just New since it returns a Controller.